tests/consensus-validation-malicious-producers.py

```diff
@@ -180,11 +180,11 @@ def stageScenario(stagedNodeInfos):
     os.makedirs(stagingDir)
     count=0
     for stagedNodeInfo in stagedNodeInfos:
-        dataPath=os.path.join(stagingDir, "tn_data_%02d" % (count))
-        os.makedirs(dataPath)
-        with open(os.path.join(dataPath, "config.ini"), "w") as textFile:
+        configPath=os.path.join(stagingDir, "etc/eosio/node_%02d" % (count))
```

Comment:
02d formats an integer (d) to a field of minimum width 2 (2), with zero-padding on the left (leading 0):